rdm/main.py

```diff
@@ -52,6 +52,7 @@ def cli(raw_arguments):
 
 def parse_arguments(arguments):
     parser = argparse.ArgumentParser(prog='rdm')
+    parser.add_argument('--version', action='version', version='v0.11.0')
```

Comment:
Nicely done! In order to ensure the version string is always up to date, we should pull it from a central spot. Ideally the version string will only be present in one place in the code base. You may want to look online to see if there are standard ways of doing this.